test/libsolidity/semanticTests/constructor/base_constructor_arguments.sol

```diff
@@ -19,5 +19,7 @@ contract Derived is Base {
         return m_a;
     }
 }
+// ====
+// bytecodeFormat: legacy,>=EOFv1
```

Comment:
Maybe we could also enable some tests that accept reference types? I was primarily interested in `calldata`, but looks like this is not allowed in a constructor. `memory` and `storage` are though.

Reply:
We should probably also think about allowing `calldata` in constructor as an EOF-only feature in the future.